slides/08-Platform-Hub/1020-LetsGetOnboarded.md

```diff
@@ -0,0 +1,11 @@
+## Let's get onboarded!
+
+Please follow these step to onboard yourself to the Platform Hub:
+
+ 1. Go to https://access-acp.digital.homeoffice.gov.uk to download and run `ACP NotProd` VPN profile.
+ 2. Go to the hub: `https://hub.acp.homeoffice.gov.uk`.
+ 3. Now click the login button and use your office 365 credentials.
```

Comment:
not seen the hub but wouldn't they have already done their o365 creds for the vpn profile?

Reply:
When login into the platform hub for the first time they'll need their o365 credential for sso. Similar to how artifactory operates.